client/src/components/Instructions/index.js

```diff
@@ -121,54 +120,41 @@ export default class Instructions extends Component {
             Create a folder for your app and enter inside.
           </div>
           <div className={styles.code}>
-            <code>
-              mkdir my-app && cd my-app
-            </code>
+            <code>mkdir my-app && cd my-app</code>
           </div>
         </div>
         <div className={styles.step}>
-          <div className={styles.instruction}>
-            Unbox the ZepKit.
-          </div>
+          <div className={styles.instruction}>Unbox the ZepKit.</div>
           <div className={styles.code}>
-            <code>
-              truffle unbox zeppelinos/zepkit
-            </code>
+            <code>truffle unbox zeppelinos/zepkit</code>
```

Comment:
The whole section has to be updated. This code is displayed at http://zepkit.zeppelinos.org.

Reply:
Sorry I don't know why I didn't catch that!